aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -528,8 +528,15 @@ Tensor linalg_solve(const Tensor& input, const Tensor& other) {
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ inverse ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+/*
+Computes the inverse of n-by-n matrix 'self'
+This is an in-place routine, content of 'self' is overriden.
```

Comment:
overriden -> overwritten. I wouldn't mention it if PR wasn't reverted, but now you might as well fix it :-)